[PyTorch][Edge] Add binarires for unittests

cccclai opened the pull request:
## Summary 
Add two models (v4 and v5) for testing runtime. (v5 will be introduced in https://github.com/pytorch/pytorch/pull/56002)

## Test plan
CI


Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#57039 [PyTorch][Edge] Add binarires for unittests**

Differential Revision: [D28047615](https://our.internmc.facebook.com/intern/diff/D28047615)

cccclai committed:
[PyTorch][Edge] Add binarires for unittests

[ghstack-poisoned]

iseeyuan commented:
1. This PR is to add two testing binary models, not removing or updating any existing model files, so it should be safe.
2. Checking in binary files are different than checking in codes. It makes sense to me to have binary files in separate PRs.

cccclai committed:
Update on "[PyTorch][Edge] Add binarires for unittests"

## Summary 
Add two models (v4 and v5) for testing runtime. (v5 will be introduced in https://github.com/pytorch/pytorch/pull/56002)

## Test plan
CI



Differential Revision: [D28047615](https://our.internmc.facebook.com/intern/diff/D28047615)

[ghstack-poisoned]
